### DISCUSSION TITLE
Adding/removing sites from data.json does nothing

### DISCUSSION THREAD

**Discussion by realbaylen** [sherlock-project/sherlock#2085] [Q&A] [answered]
I have tried adding/removing entries from data.json but Sherlock does not check the newly added ones and it continues to check the removed ones. The only thing that is seemingly affected by the data.json file is the sites_list.py to generate the site list markdown file. Thanks

**Comment by matheusfelipeog** [comment, marked as answer]
@realbaylen You should use the `--local` flag to force Sherlock to use the local `data.json` file. This way, your changes will take effect locally.
